cmake/projects/x264/schemes/url_sha1_x264.cmake.in

```diff
@@ -0,0 +1,341 @@
+      -P
+      "@HUNTER_PACKAGE_SOURCE_DIR@/universal/${merge_lipo_script}"
+  )
+  set(ios_built_arch_roots)
+  set(multi_arch_install_root "@HUNTER_PACKAGE_BUILD_DIR@/multi-arch")
+  foreach(ios_architecture ${ios_architectures})
+    hunter_status_debug("Autotools: building for iOS architecture ${ios_architecture}")
+    #clear the conf options
+    set(is_simulator FALSE)
+    if(${ios_architecture} STREQUAL "armv7"
+        OR ${ios_architecture} STREQUAL "armv7s")
+      set(configure_host "arm-apple-darwin")
+    elseif(${ios_architecture} STREQUAL "arm64")
+      set(configure_host "aarch64-apple-darwin")
+    elseif(${ios_architecture} STREQUAL "i386")
+      set(configure_host "i386-apple-darwin")
+      set(is_simulator TRUE)
+    elseif(${ios_architecture} STREQUAL "x86_64")
+      set(configure_host "x86_64-apple-darwin")
+      set(is_simulator TRUE)
+    else()
+      hunter_fatal_error("iOS architecture: ${ios_architecture} not supported")
+    endif()
+
+    set(arch_flags)
+    # Extra space at the end of the arch_flags is needed below when appending
+    # to configure_opts, please do not remove!
+    if(is_simulator)
+      set(arch_flags "-arch ${ios_architecture} -isysroot ${IPHONESIMULATOR_SDK_ROOT} -miphoneos-version-min=${IOS_SDK_VERSION} ")
+    else()
+      set(arch_flags "-arch ${ios_architecture} -isysroot ${IPHONEOS_SDK_ROOT} -miphoneos-version-min=${IOS_SDK_VERSION} ")
+    endif()
+
+    # architecture specific source dir
+    set(arch_source_dir
+      "@HUNTER_PACKAGE_SOURCE_DIR@/multi-arch-build/${ios_architecture}"
+    )
+    set(arch_target
+        "${target_name}-${ios_architecture}"
+    )
+    set(arch_install_dir
+        "${multi_arch_install_root}/${ios_architecture}"
+    )
+
+    set(configure_command
+        ${shell_env}
+        ${x264_configure_command}
+        "--host=${configure_host}"
+        "--extra-cflags=${arch_flags}${cflags}"
+        "--extra-ldflags=${arch_flags}${ldflags}"
+        "--prefix=${arch_install_dir}"
+    )
+    ExternalProject_Add(${arch_target}
+        URL
+          @HUNTER_PACKAGE_URL@
+        URL_HASH
+          SHA1=@HUNTER_PACKAGE_SHA1@
+        DOWNLOAD_DIR
+          "@HUNTER_PACKAGE_DOWNLOAD_DIR@"
+        SOURCE_DIR
+          ${arch_source_dir}
+        INSTALL_DIR
+          ${arch_install_dir}
+          # not used, just avoid creating Install/<name> empty directory
+        CONFIGURE_COMMAND
+          ${configure_command}
+        BUILD_COMMAND
+          ${build_command}
+        BUILD_IN_SOURCE
+          1
+        INSTALL_COMMAND
+          ${install_command}
+    )
+
+    list(APPEND ios_built_arch_roots ${arch_install_dir})
+    add_dependencies(
+      "${ios_universal_target}"
+      "${arch_target}"
+    )
+  endforeach()
+
+  set(HUNTER_PACKAGE_INSTALL_PREFIX "@HUNTER_PACKAGE_INSTALL_PREFIX@")
+  configure_file(
+      "@HUNTER_GLOBAL_SCRIPT_DIR@/${merge_lipo_script}.in"
+      "@HUNTER_PACKAGE_SOURCE_DIR@/universal/${merge_lipo_script}"
```

Comment:
Aye Caramba, actually these packages are not working on iOS upstream, I had code to locally treat `*.pc` files, which I don't think I sent upstream :S 

We had a problem with unrelocatable files during the merge, `*.pc` text files had in them the directory of the install prefix, so the `autotools-merge-lipo` script [ https://github.com/ruslo/hunter/blob/bfdc19731a5e467674abd7aef9a227678c4176a6/scripts/autotools-merge-lipo.cmake.in ] would fail merging them.

By looking at the script now, I don't think I uploaded the logic to hunter repo.

Sorry about the confusion, this was done some time ago. I had a prototype of an app using `odb-sqlite` on iOS hence the confusion, I don't think these packages are working with the upstream version of hunter.